### DISCUSSION TITLE
How can connect with socket.io

### DISCUSSION THREAD

**Discussion by MohMehrnia** [jchristn/WatsonWebsocket#147] [Q&A]
How can connect with socket.io?

**Comment by jchristn** [comment]
I don't have experience with socket.io, but found this, which may be helpful: https://stackoverflow.com/questions/22232023/can-i-use-socket-io-client-to-connect-to-a-standard-websocket

Moving this to discussions.

**Reply by MohMehrnia** [reply]
I read this, but i tried socket.io can not connected
